Framework/PythonInterface/mantid/geometry/CMakeLists.txt

```diff
@@ -53,7 +53,9 @@ create_module(${MODULE_TEMPLATE} ${MODULE_DEFINITION} ${EXPORT_FILES})
 
 set(SRC_FILES)
 
-set(INC_FILES)
+set(INC_FILES
+  inc/MantidPythonInterface/geometry/ComponentInfoPythonIterator.h
```

Comment:
These 2 are never used outside the `PythonInterface/geometry` module. They were included in the `api` which now failed as `api` is dependent on `geometry`, so they weren't available in the includes.